core/controllers/library_test.py

```diff
@@ -195,12 +195,12 @@ def test_thumbnail_icons_exist_for_each_category(self):
         # Test that an icon exists for each default category.
         for category in all_categories:
             utils.get_file_contents(os.path.join(
-                'static', 'images', 'subjects',
+                'static', utils.get_cache_slug(), 'images', 'subjects',
```

Comment:
So, for windows the cache_slug that is needed in html would be a forward slash and file paths would be a backward slash. Okay, so I'll define two cache slug constants then: one for file paths and the other for html files.